### PR TITLE
sci-misc/kaldi: Version bump

### PR DESCRIPTION
**Pull request by akreal**
Adopt the version scheme introduced by upstream
Add double-precision USE flag

### CODE REVIEW

File: sci-misc/kaldi/kaldi-5.1.30.ebuild [hunk 1]
```diff
@@ -1,102 +1,123 @@
...
-inherit eutils toolchain-funcs cuda flag-o-matic
+inherit toolchain-funcs cuda flag-o-matic
 
+COMMIT_ID="828f0b9cc6bbf2ad412c17345a66a5437fc95cd2"
 DESCRIPTION="A toolkit for speech recognition"
 HOMEPAGE="http://kaldi-asr.org/"
-SRC_URI="http://gentoo.akreal.net/distfiles/${P}.tar.xz"
+SRC_URI="https://github.com/kaldi-asr/kaldi/archive/${COMMIT_ID}.tar.gz -> ${P}.tar.gz"
+
+S=${WORKDIR}/${PN}-${COMMIT_ID}/src
```
Comment (akreal) [inline sci-misc/kaldi/kaldi-5.1.30.ebuild]: Done.

File: sci-misc/kaldi/kaldi-5.1.30.ebuild [hunk 2]
```diff
@@ -1,102 +1,123 @@
...
 }
 
 src_prepare() {
-	epatch \
-		"${FILESDIR}"/Makefile.patch \
-		"${FILESDIR}"/default_rules.mk.a7d9824.patch
+	eapply \
+		"${FILESDIR}"/Makefile.0d5e4b1.patch \
+		"${FILESDIR}"/default_rules.mk.96eec2b.patch
+	eapply_user
```
Comment (akreal) [inline sci-misc/kaldi/kaldi-5.1.30.ebuild]: Done.

File: sci-misc/kaldi/kaldi-5.1.30.ebuild [hunk 3]
```diff
@@ -1,102 +1,123 @@
...
+	use test || append-cxxflags -DNDEBUG
 
-	if [[ ${cblas_provider} =~ atlas ]]; then
-		append-cxxflags -DHAVE_ATLAS
-	elif [[ ${cblas_provider} =~ mkl ]]; then
-		append-cxxflags -DHAVE_MKL
-	elif [[ ${cblas_provider} =~ openblas ]]; then
-		append-cxxflags -DHAVE_OPENBLAS $($(tc-getPKG_CONFIG) --cflags lapacke)
+	if use speex; then
+		append-cxxflags -DHAVE_SPEEX
```
Comment (akreal) [inline sci-misc/kaldi/kaldi-5.1.30.ebuild]: Changed.

File: sci-misc/kaldi/kaldi-5.1.65.ebuild [hunk 4]
```diff
@@ -1,102 +1,124 @@
...
 LICENSE="Apache-2.0"
 SLOT="0"
-IUSE="doc test cuda"
 KEYWORDS="~amd64"
+IUSE="cuda doc double-precision speex test"
+REQUIRED_USE="double-precision? ( !speex )"
 
 RDEPEND="
-	media-libs/speex
+	speex? ( media-libs/speex )
```
Comment (jlec) [inline sci-misc/kaldi/kaldi-5.1.65.ebuild]: Please move all deps based on USE to the end.

Comment (akreal) [inline sci-misc/kaldi/kaldi-5.1.65.ebuild]: Fixed.

**Pull request merged**